modules/controlled_access_terms_defaults/config/install/core.entity_form_display.taxonomy_term.resource_types.default.yml

```diff
@@ -0,0 +1,58 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.field.taxonomy_term.resource_types.field_external_authority_link
+    - taxonomy.vocabulary.resource_types
+  module:
```

Comment:
Should also include a dependency on controlled_access_terms.

Reply:
Added - see line below this.